grunt/config/uglify.js

```diff
@@ -1,26 +1,22 @@
-module.exports = function (grunt, options) {
-    return {
-        minify: {
-            files: [
-                {
-                    expand: true,
-                    cwd: '<%= outputdir %>libraries/',
-                    src: ['*.js', '!*.min.js'],
-                    dest: '<%= outputdir %>libraries/'
-                },
-                {
-                    expand: true,
-                    cwd: '<%= outputdir %>/adapt/js',
-                    src: ['*.js', '!*.min.js'],
-                    dest: '<%= outputdir %>/adapt/js'
-                },
-                {
-                    expand: true,
-                    cwd: '<%= outputdir %>',
-                    src: ['*.js', '!*.min.js'],
-                    dest: '<%= outputdir %>'
-                }
-            ]
-        }
+module.exports = function(grunt, options) {
+  return {
+    minify: {
+      files: [{
+        expand: true,
+        cwd: '<%= outputdir %>libraries/',
+        src: ['*.js', '!*.min.js'],
+        dest: '<%= outputdir %>libraries/'
+      }, {
```

Comment:
@chetan-hajare - Could you put the opening '{' and associated closing '}' on a new line please?